src/server/router/application.ts

```
@@ -184,34 +155,216 @@ export const applicationRouter = router({
+            formYear: formSubmission.formYear,
+            submitterId: formSubmission.submitterId,
+          },
         },
+        update: formSubmission,
+        create: formSubmission,
+      });
+
+      let gradDate = null;
+      if (input.studyExpectedGraduation) {
+        const possible = new Date(input.studyExpectedGraduation);
+        if (!isNaN(possible.getTime())) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)